source/Camera/InfoUtilOther/GetCameraIntrinsics/Properties/AssemblyInfo.cs

```diff
@@ -31,6 +31,7 @@
 //
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
-// [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+
+[assembly: AssemblyVersion("2.0.0")]
```

Comment:
Actually I think these should be 1.0. They don't need to follow the SDK. It will be more confusing if its 2.0, and then we update to SDK 2.1, and wonder if these should be updated as well. But maybe these files can be removed completely without any issues.

Reply:
I was thinking that is would be confusing that it is 1.0 and we just released 2.0. I will try to remove it.

Reply:
done

Reply:
![image](https://user-images.githubusercontent.com/46567886/83681650-fcc23780-a5e2-11ea-81fa-5cb04207e2fa.png)
:/

Reply:
Switch to 1 or keep this or try to disable warning?

Reply:
I think I would just set all to 1.0.